### ISSUE TITLE
Reproduce issue

### ISSUE THREAD

**Issue by wpf89928** [tienmanhptit1312/Disentangle-VAE-for-VC#7]
A good work! 

Just want to ask if the script is the latest one, becasue I tried to train from scratch but fail to convert. Are the hyper-parameters the ones used to produce the provided model? How many epochs did you run? I notice that you selected the 1560 epochs model, is there any way to guide the model selection? 

Thank you!

**Comment by v-manhlt3** [comment]
Hi, the hyperparameters are the ones used to train my provided model. The pretrained model is trained about 16,000 epochs, but I chose the best one. Could you show me your training log to debug?

**Comment by wpf89928** [comment]
> 

Hi, thanks for your reply! I use a different batch size (512). I am running with your default batch size (8). 

BTW, you use the epoch 1650 model, how frequent you save the checkpoint? In the 'training.sh' script, the 'report-interval=500' .

**Comment by v-manhlt3** [comment]
I think a large batch size is the problem. The reason is that Libri dataset has some noisy samples, so with the large batch size this can cause gradient explosion/vanishing. If you want to train my model with a large batch size, you can use the gradient clipping method.

I evaluated and saved my model every epoch.

**Comment by wpf89928** [comment]
Thanks! I am running the experiment with the default batch size. Will approach you if I have more questions.

**Comment by wpf89928** [comment]
hi, I tried the batch size 8, and obtained the model of epoch 1100. It still does not work...May I know from approximately what epochs the model can succeed in converting? 

Btw, in the training.sh script,  --style_cof appears twice.
